src/makeRobot.js

```diff
@@ -38,7 +38,91 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      const infoName = `name: ${this.name}, `;
+      const infoChip = `chip version: ${this.version}, `;
+      const infoWheels = `wheels: ${this.wheels}`;
+
+      return infoName + infoChip + infoWheels;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+    goForward(value) {
+      if (!value) {
+        this.coords.y++;
+      }
+
+      if (value > 0) {
+        this.coords.y += value;
+      }
+
+      if (value <= 0) {
+        this.coords.y += 0;
+      }
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(value = 1) {
      if (value > 0) {
        this.coords.y += value;
      }

      return this;
    },
```
You can add default value in brackets. 
In that case only one check will be enough
Please fix this for all moving methods